assets/custom-icons/_data/custom-icons.json

```diff
@@ -30,6 +30,10 @@
     {
       "title": "Bitwarden"
     },
+    {
+      "title": "Brave Creators",
+      "slug": "brave"
```

Comment:
This should be the name of the file, so `brave_creators`

Reply:
I have to rename SVG files or rename the title in JSON ?
This is my first time contributing in an opensource project.

Reply:
Please refer to [the documentation](https://github.com/ente-io/auth/blob/main/CONTRIBUTING.md#icons).

If the name of the SVG file is different from the `title`, we rely on `slug`. Since this is the case here, the `slug` should be the name of the SVG file - `brave_creators`

Reply:
updated slugs